docs/NooBaaNonContainerized/Logging.md

```diff
@@ -0,0 +1,86 @@
+# NooBaa Non Containerized - Logging
+
+1. [Introduction](#introduction)
+2. [NooBaa Service logs](#noobaa-service-logs)
+    1. [Journal logs](#journal-logs)
+    2. [Rsyslog](#rsyslog) 
+        1. [Rsyslog status check](#rsyslog-status-check)
+        2. [Logrotate](#logrotate)
+            1. [Logrotate automatic rotation](#logrotate-automatic-rotation)
+            1. [Manual Logrotate trigerring](#manual-logrotate-triggering)
+4. [NooBaa Debug levels](#noobaa-debug-levels)
+    1. [Supported log debug levels](#supported-log-debug-levels)
+    2. [Debug level Increase](#debug-level-increase)
+5. [NooBaa Logs format](#noobaa-logs-format)
+
+## Introduction
+
+This document provides an overview of the logging capabilities within NooBaa, detailing how to configure and utilize logs for monitoring and troubleshooting.
+
+## NooBaa Service Logs
+
+NooBaa service logs are written directly to stderr and to syslog (will be deprecated in the future).
+1. NooBaa writes the logs into stderr and to /var/log/noobaa.log.
+2. The journalctl service watches stderr and writes it to journal.
+3. A syslog service, like rsyslog/syslog_ng watches journal and writes it to /var/log/messages/.
+
+### Journal logs
+1. NooBaa service logs -
+
+Run the following command in order to display the NooBaa service logs -
+
+```sh
+journalctl -u noobaa.service
+```
+
+### Rsyslog
+
+NooBaa logs are configured using rsyslog and logrotate. NooBaa RPM will configure rsyslog and logrotate if both are already running.
+
+#### Rsyslog status check
+```
+systemctl status rsyslog
+```
+
+NooBaa logs are pushed to `var/log/noobaa.log` and the log is rotated and compressed daily.
+
+The following files contain NooBaa specific configuration for rsyslog and logrotate - 
+1. `etc/rsyslog.d/noobaa_syslog.conf`
+2. `etc/logrotate.d/noobaa/logrotate_noobaa.conf` 
+
+Verify the rsyslog and logrotate rpm configuration is complete by checking the files above.
+
+#### Logrotate
+
+##### Logrotate automatic rotation
+
+- Logrotate configuration is set up under `/etc/logrotate.d/noobaa/`. 
+- In order to trigger rotation when log files reaches size threshold, each log file (`/var/log/noobaa.log` and `/var/log/noobaa_events.log`) uses its own rsyslog `outchannel`.
+- Logrotate is triggered when file size reaches 105MB limit. 
```

Comment:
What is the source of the number 105MB? can we refer to it from the config file?